src/main/java/sc/iview/SciView.kt

```diff
@@ -1269,8 +1279,10 @@ class SciView : SceneryBase, CalibratedRealInterval<CalibratedAxis> {
      * @param <T> pixel type of image
      * @return a Node corresponding to the Volume
     </T> */
+    @JvmName("addVolume1")
```

Comment:
The necessity of `@JvmName` might indicate there's another function doing exactly the same, with almost the same parameters. Could you please check that? That might have escaped the Java to Kotlin conversion.

Reply:
Will do!

Reply:
I addressed this and was hopefully able to keep the correct signatures.